[A770] language/test_core.py::test_noinline[shared] failed

```
2024-05-09T19:34:10.8294846Z FAILED language/test_core.py::test_noinline[shared] - AssertionError: assert False
2024-05-09T19:34:10.8305399Z  +  where False = <built-in method equal of type object at 0x7f54f8a1b600>(tensor([[nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan,\n         nan, nan],\n        [nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan,\n         nan, nan],\n        [nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan,\n         nan, nan],\n        [nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan,\n         nan, nan],\n        [99., 99., 99., 99., 99., 99., 99., 99., 99., nan, 99., 99., 99., nan,\n         99., 99.],\n        [99., 99., 99., 99., 99., 99., 99., 99., 99., nan, 99., 99., 99., nan,\n         99., 99.],\n        [99., 99., 99., 99., 99., 99., 99., 99., 99., nan, 99., 99., 99., nan,\n         99., 99.],\n        [99., 99., 99., 99., 99., 99., 99., 99., 99., nan, 99., 99., 99., nan,\n         99., 99.],\n        [nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan,\n         nan, nan],\n        [nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan,\n         nan, nan],\n        [nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan,\n         nan, nan],\n        [nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan,\n         nan, nan],\n        [99., 99., 99., 99., 99., 99., 99., 99., 99., nan, 99., 99., 99., nan,\n         99., 99.],\n        [99., 99., 99., 99., 99., 99., 99., 99., 99., nan, 99., 99., 99., nan,\n         99., 99.],\n        [99., 99., 99., 99., 99., 99., 99., 99., 99., nan, 99., 99., 99., nan,\n         99., 99.],\n        [99., 99., 99., 99., 99., 99., 99., 99., 99., nan, 99., 99., 99., nan,\n         99., 99.]], device='xpu:0'), ((tensor([[16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.],\n        [16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16., 16.,\n         16., 16.]], device='xpu:0') + tensor([1.], device='xpu:0')) + tensor([2.], device='xpu:0')))
2024-05-09T19:34:10.8313484Z  +    where <built-in method equal of type object at 0x7f54f8a1b600> = torch.equal
```

--- a/scripts/skiplist/a770/language.txt
+++ b/scripts/skiplist/a770/language.txt
@@ -14,7 +14,6 @@ python/test/unit/language/test_core.py::test_dot3d[4-4-32-32-32-32-32-float64-fl
 python/test/unit/language/test_core.py::test_dot3d[4-8-32-32-32-32-32-float64-float64]
 python/test/unit/language/test_core.py::test_dot3d[4-16-32-32-32-32-32-float64-float64]
 # https://github.com/intel/intel-xpu-backend-for-triton/issues/983
-python/test/unit/language/test_core.py::test_noinline[shared]
 # test_dot
 python/test/unit/language/test_core.py::test_dot[r".*-float64-1-None"]@regexp
 python/test/unit/language/test_core.py::test_dot[r"1-64-128-128-2-.*"]@regexp

--- a/scripts/skiplist/arl-h/language.txt
+++ b/scripts/skiplist/arl-h/language.txt
@@ -7,8 +7,6 @@ python/test/unit/language/test_core.py::test_dot3d[8-2-32-32-32-8-16-float16-flo
 python/test/unit/language/test_core.py::test_dot3d[8-4-32-32-32-8-16-float16-float16]
 python/test/unit/language/test_core.py::test_dot3d[8-1-32-32-32-8-16-float16-float16]
 # https://github.com/intel/intel-xpu-backend-for-triton/issues/983
-python/test/unit/language/test_core.py::test_noinline[shared]
-
 # test_dot
 python/test/unit/language/test_core.py::test_dot[r"1-128-128-64-2-.*"]@regexp
 python/test/unit/language/test_core.py::test_dot[r"1-64-128-128-2-.*"]@regexp

--- a/scripts/skiplist/arl-s/language.txt
+++ b/scripts/skiplist/arl-s/language.txt
@@ -7,7 +7,6 @@ python/test/unit/language/test_core.py::test_dot3d[8-2-32-32-32-8-16-float16-flo
 python/test/unit/language/test_core.py::test_dot3d[8-4-32-32-32-8-16-float16-float16]
 python/test/unit/language/test_core.py::test_dot3d[8-1-32-32-32-8-16-float16-float16]
 # https://github.com/intel/intel-xpu-backend-for-triton/issues/983
-python/test/unit/language/test_core.py::test_noinline[shared]
 # test_dot
 python/test/unit/language/test_core.py::test_dot[r"1-64-128-128-2-.*"]@regexp
 python/test/unit/language/test_core.py::test_dot[r"1-128-128-64-2-.*"]@regexp

--- a/scripts/skiplist/mtl/language.txt
+++ b/scripts/skiplist/mtl/language.txt
@@ -5,7 +5,6 @@ python/test/unit/language/test_core.py::test_dot3d[r".*-float16-float32"]@regexp
 python/test/unit/language/test_core.py::test_dot3d[r".*-32-32-float16-float16"]@regexp
 
 # https://github.com/intel/intel-xpu-backend-for-triton/issues/983
-python/test/unit/language/test_core.py::test_noinline[shared]
 python/test/unit/language/test_core.py::test_dot_max_num_imprecise_acc[r".*-128-128-256-256"]@regexp
 python/test/unit/language/test_block_pointer.py::test_block_ptr_matmul_no_scf
 python/test/unit/language/test_pipeliner.py::test_pipeline_matmul[True]

